certified-connectors/Powell Teams/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Powell.Teams.Web.PublicAPI",
+    "description": "Powell Teams API endpoints documentation.",
+    "version": "V1"
+  },
```

Comment:
@PowellSoftware Please add metadata to your connector.

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)